codex-ui/src/vue/components/alert/AlertTransition.vue

```diff
@@ -0,0 +1,35 @@
+<template>
+  <transition-group
+    tag="div"
+    name="fade"
+    @leave="leave"
+  >
+    <slot />
+  </transition-group>
+</template>
+
+<script setup lang="ts">
+
+const leave = (el: unknown) => {
+  if (el instanceof HTMLElement) {
+    el.style.left = el.offsetLeft + 'px';
+    el.style.top = el.offsetTop + 'px';
+    el.style.width = getComputedStyle(el).width;
+    el.style.position = 'absolute';
+  }
+};
```

Comment:
do not js for simple animations, please, especially when it leads Layout. It is bad for performance reasons.

Reply:
effected and removed